release-notes/8.0/8.0.1/api-diff/Microsoft.NETCore.App/8.0.1_System.Runtime.Intrinsics.Arm.md

```diff
@@ -0,0 +1,1328 @@
+# System.Runtime.Intrinsics.Arm
+
+``` diff
+ namespace System.Runtime.Intrinsics.Arm {
```

Comment:
Arm intrinsics LGTM. This is jsut a lot of `ConstantExpected` attributes being added.